supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiRuntimeAttributes.scala

```diff
@@ -1,271 +1,17 @@
-      Option(
-        GpuResource(gpuType.getOrElse(GpuType.DefaultGpuType),
-                    gpuCount.getOrElse(GpuType.DefaultGpuCount),
-                    gpuDriver.getOrElse(GpuResource.DefaultNvidiaDriverVersion)
-        )
-      )
-    } else {
-      None
-    }
-
-    val zones: Vector[String] = RuntimeAttributesValidation.extract(ZonesValidation, validatedRuntimeAttributes)
-    val preemptible: Int =
-      RuntimeAttributesValidation.extract(preemptibleValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val bootDiskSize: Int =
-      RuntimeAttributesValidation.extract(bootDiskSizeValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val memory: MemorySize =
-      RuntimeAttributesValidation.extract(memoryValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val disks: Seq[PipelinesApiAttachedDisk] =
-      RuntimeAttributesValidation.extract(disksValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val docker: String = RuntimeAttributesValidation.extract(dockerValidation, validatedRuntimeAttributes)
-    val failOnStderr: Boolean =
-      RuntimeAttributesValidation.extract(failOnStderrValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val continueOnReturnCode: ContinueOnReturnCode = RuntimeAttributesValidation.extract(
-      continueOnReturnCodeValidation(runtimeAttrsConfig),
-      validatedRuntimeAttributes
-    )
-    val noAddress: Boolean =
-      RuntimeAttributesValidation.extract(noAddressValidation(runtimeAttrsConfig), validatedRuntimeAttributes)
-    val useDockerImageCache: Option[Boolean] = RuntimeAttributesValidation.extractOption(
-      useDockerImageCacheValidation(runtimeAttrsConfig).key,
-      validatedRuntimeAttributes
-    )
-
-    new PipelinesApiRuntimeAttributes(
-      cpu,
-      cpuPlatform,
-      gpuResource,
-      zones,
-      preemptible,
-      bootDiskSize,
-      memory,
-      disks,
-      docker,
-      failOnStderr,
-      continueOnReturnCode,
-      noAddress,
-      googleLegacyMachineSelection,
-      useDockerImageCache,
-      checkpointFileName
-    )
-  }
-}
-
-object ZonesValidation extends RuntimeAttributesValidation[Vector[String]] {
-  override def key: String = PipelinesApiRuntimeAttributes.ZonesKey
-
-  override def coercion: Iterable[WomType] = Set(WomStringType, WomArrayType(WomStringType))
-
-  override protected def validateValue: PartialFunction[WomValue, ErrorOr[Vector[String]]] = {
-    case WomString(s) => s.split("\\s+").toVector.validNel
-    case WomArray(womType, value) if womType.memberType == WomStringType =>
-      value.map(_.valueString).toVector.validNel
-  }
-
-  override protected def missingValueMessage: String =
-    s"Expecting $key runtime attribute to be either a whitespace separated String or an Array[String]"
-}
-
 object DisksValidation extends RuntimeAttributesValidation[Seq[PipelinesApiAttachedDisk]] {
```

Comment:
I think this same `DisksValidation` exists in Batch, can we just change the import to have TES use that one, and get rid of this file?